src/app/pages/map-view/sidebar-section/sidebar-section.component.html

```diff
@@ -0,0 +1,39 @@
+<gd-sidebar-header [title]="cards[0].title" [(isCollapsed)]="isCollapsed" class="sidebar-header">
+</gd-sidebar-header>
+
+<!-- actions-bar -->
+<div *ngIf="cards[0].model!=='lm'&&cards[0].model!=='potsdam'" class="actions-bar">
+    <!-- group-actions -->
+    <div class="group-actions">
+        <button gdActionBtn [(expanded)]="expanded" [action]="'analyze'" (expandedChange)="onExpandedChange()" [class]="'action icon-'+ 'analyze'">
+                        Analyze Area
+            </button>
+    </div>
+    <!-- end of group-actions -->
+    <!-- action-panel group-->
+    <!-- action-panel -->
+    <div class="action-panel" [class.-expanded]="expanded=='analyze'">
```

Comment:
Saw this in a few places, but noticed it here most. Continue to respect standard JS spacing conventions in your Angular templates to improve readability, eg:
```
*ngIf="cards[0].model!=='lm'&&cards[0].model!=='potsdam'"
```
becomes
```
*ngIf="cards[0].model !== 'lm' && cards[0].model !== 'potsdam'"
```

I haven't seen this as part of the official style guide, but I also haven't ever seen statements squashed together in good example code.